observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -0,0 +1,91 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-CLF_{context}"]
+= Configuring log forwarding
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` CR serves as a single point of configuration for log forwarding, making it easier to manage and maintain log collection and forwarding rules.
+
```

Comment:
we can place pods on specific node or limit resources using .spec.collector.
For example:
spec:
  collector:
    nodeSelector:
      kubernetes.io/os: "linux"
    resources: {}

Reply:
Do we need to add the [above information](https://github.com/openshift/openshift-docs/pull/78477/files#r1687613699) in _Configuring log forwarding_ section? 
(Need clarity because this is just the introductory section.)

Reply:
The above information will be covered in Multi log forwarder CLF.

Reply:
> Multi log forwarder CLF

For 6.0 we should reconsider the user of "multi log forwarder CLF" as starting in 6.0.  CLF becomes like any other custom resource on the cluster in that there can always be "multiple".  There is no need to distinguish it separately from our legacy usecases